### DISCUSSION TITLE
CloudFlare patched chromedriver

### DISCUSSION THREAD

**Discussion by duylmaoxd** [ultrafunkamsterdam/undetected-chromedriver#1594] [General]
recently CloudFlare has patched chromedriver , pls update

**Comment by henzycuong1** [comment]
does anyone recomend any librare alternative ?

**Reply by max32002** [reply]
https://github.com/g1879/DrissionPage

**Comment by krypterro** [comment]
What do you mean Cloudflare patched Chromedriver, what does Cloudflare have to do with Google in that respect?

**Reply by codsec** [reply]
It means CloudFlare is detecting it anyway, no matter what we do xd(for now at least)

**Comment by airparo23** [comment]
Is it working now? Can somebody please update.

**Reply by duylmaoxd** [reply]
it's working, but if u run it many times without proxies, u will be stuck at cloudflare capcha and cant verify

**Reply by welinkito** [reply]
is it working when you click on a cloudflare checkbox? which is usually shown  when you are performing an action on the site... After the first cloudflare check when you openning the site.

**Comment by LeadhuntAI** [comment]
For me its not working, tried with private residential proxy and without. Both get cought. 
For those that can use it right now, do you do any additional options/settings when you spin up the driver?

**Comment by pokemaster974** [comment]
Got a similar answer from FlareSolverr contributor here : https://github.com/FlareSolverr/FlareSolverr/issues/951
